### DISCUSSION TITLE
Is there any mature solution for hot update runtime bundle？

### DISCUSSION THREAD

**Discussion by SnowingFox** [obytes/react-native-template-obytes#416] [Q&A]
I try to use custom-expo-updates-sever，but it's just a server api, so i want to know is there any solution for hot update runtime bundle?

**Comment by yjose** [comment]
Hey @SnowingFox, I'm not sure, to be honest. The best place to ask this question is probably in the repo.https://github.com/expo/custom-expo-updates-server

**Reply by SnowingFox** [reply]
@yjose i know and tried this repo before, but expo-updates can't update bundle by incremental updating which is based on git diff, it will take a lot of network bandwidth
